src/test/java/pages/BasePage.java

```diff
@@ -428,5 +438,83 @@ protected boolean isAlertDisplayed() {
             return false;
         }
     }
+
+    protected String getSaltString10() {
+        String SALTCHARS = "AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz";
```

Comment:
String SALTCHARS = change to regex

Reply:
Removed couple of unused methods with SaltString

Reply:
I don't know how to make it. If you have proposition I'm opened for it)